examples/rng/device/include/rng_example_helper.hpp

```diff
@@ -30,7 +30,7 @@ struct has_member_code_meta<T, std::void_t<decltype(std::declval<T>().get_multi_
 template <typename T, typename std::enable_if<has_member_code_meta<T>::value>::type* = nullptr>
 auto get_multi_ptr(T acc) {
 // Workaround for AdaptiveCPP, as they do not yet support the get_multi_ptr function
-#ifndef __HIPSYCL__
+#ifndef __ADAPTIVECPP__
```

Comment:
Could be out-of-scope for this PR, but since we anyway require ACpp 24.06, this workaround (here and in other files) can be removed.